metadata-ingestion/tests/integration/sql-queries/test_sql_queries.py

```diff
@@ -0,0 +1,78 @@
+import os
+import pathlib
+
+import pytest
+import requests
+import yaml
+
+from datahub.ingestion.run.pipeline import Pipeline
+from datahub.testing import mce_helpers
+from tests.test_helpers import fs_helpers
+from tests.test_helpers.docker_helpers import wait_for_port
+
+
+def check_mockserver_health():
+    """Custom health check for MockServer using /health endpoint."""
+    try:
+        response = requests.get("http://localhost:8080/health", timeout=2)
+        return response.status_code == 200
+    except Exception:
+        return False
+
+
+@pytest.fixture(scope="module", autouse=True)
+def docker_datahub_service(docker_compose_runner, pytestconfig):
+    """Start Docker mock DataHub service for all tests."""
+    test_resources_dir = pytestconfig.rootpath / "tests/integration/sql-queries"
+
+    with docker_compose_runner(
+        test_resources_dir / "docker-compose.yml", "datahub-mock", cleanup=True
+    ) as docker_services:
+        wait_for_port(
+            docker_services,
+            container_name="datahub-mock",
+            container_port=8080,
+            timeout=60,
+            checker=check_mockserver_health,
```

Comment:
running a mock server in a docker container seems weird - why can't we just mock the schema resolver's fetch methods to return the right data?

also, how did you generate the expectations file?